cli/README.md

```diff
@@ -0,0 +1,46 @@
+# AMP-Toolbox CLI
+
+A command line interface (CLI) for AMP-Toolbox
+
+Supported commands:
+- [update-cache](#update-cache): CLI for the [AMP update-cache](https://developers.google.com/amp/cache/update-cache).
```

Comment:
suggestion: `updates documents stored in [AMP Caches](https://developers.google.com/amp/cache/update-cache).`